<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sensorCompare</class>
 <widget class="QDialog" name="sensorCompare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compare sensors</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Compare</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Sensor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sensorList"/>
            </item>
            <item>
             <widget class="QPushButton" name="loadSensor">
              <property name="text">
               <string>Load sensor</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="checkQuality">
                <property name="text">
                 <string>Quality</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkTime">
                <property name="text">
                 <string>filter</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QFrame" name="compareFrame">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Begin position</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateTimeEdit" name="beginPos">
                   <property name="displayFormat">
                    <string>yyyy-MM-dd hh:mm:ss</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>End position</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateTimeEdit" name="endPos">
                   <property name="displayFormat">
                    <string>yyyy-MM-dd hh:mm:ss</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2,1,5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Remove Sensor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="selectedSensor"/>
         </item>
         <item>
          <widget class="QPushButton" name="removeSensor">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="chartFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollContent">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>842</width>
               <height>465</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="margin">
               <number>3</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>frame</zorder>
      <zorder></zorder>
      <zorder>chartFrame</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Help</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600;&quot;&gt;Help&lt;/span&gt;&lt;/p&gt;
&lt;hr /&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600;&quot;&gt;Compare sensor &lt;/span&gt;&lt;/p&gt;
&lt;hr /&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt;&quot;&gt;Through this interface it is possible to plot several Sensors in the same window to visually compare them:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;A number of sensors can be selected from the drop-down menu and added to the display window with the Load sensor button. If the quality check-box is selected only the quality data is loaded. If the filter option is checked the user have to specify a time interval for which sensor data should be loaded, e.g. when two Sensor to be compared do not have the same starting or ending dates.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Any Sensor loaded into the display can be removed by selecting it from the Remove Sensor drop-down menu and pressing the Remove button.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If required, axes and plotting options can be formatted using the inbuilt matplotlib tools&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
